options.go

```diff
@@ -319,6 +320,26 @@ func Resolvers(resolvers ...Resolver) Option {
 	})
 }
 
+// IgnoreFieldsRegex will cause kong.New() to skip field names that match any
+// of the provided regex patterns. This is useful if you are not able to add a
+// kong="-" struct tag to a struct/element before the call to New.
+//
+// Example: When referencing protoc generated structs, you will likely want to
+// ignore/skip XXX_* fields.
+func IgnoreFieldsRegex(regexes ...*regexp.Regexp) Option {
```

Comment:
Please change this accept strings, and compile the regexes in the OptionFunc. It would be tedious to require users to compile their patterns.

Reply:
Sure thing!